From 69f8b5ad0467b1f305a5bb753953061e7998d3dc Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 21 Apr 2014 14:19:02 +0000
Subject: [PATCH] tools: fix headers for bcm43xx

PATH_MAX is defined in limits.h so includ it explicitly.

This fixes the following compile error on musl libc (Alpine Linux):
tools/hciattach_bcm43xx.c: In function 'bcm43xx_locate_patch':
tools/hciattach_bcm43xx.c:313:14: error: 'PATH_MAX' undeclared (first use in this function)
    char path[PATH_MAX];
                  ^

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 tools/hciattach_bcm43xx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/hciattach_bcm43xx.c b/tools/hciattach_bcm43xx.c
index ad9b239..cb4bfb9 100644
--- a/tools/hciattach_bcm43xx.c
+++ b/tools/hciattach_bcm43xx.c
@@ -34,6 +34,7 @@
 #include <errno.h>
 #include <dirent.h>
 #include <time.h>
+#include <limits.h>
 
 #include <bluetooth/bluetooth.h>
 #include <bluetooth/hci.h>
-- 
1.9.2

